### PR TITLE
Chore/api/improve jest ci workflow part 2+alicia suggestions

### PR DESCRIPTION
**Pull request by aagm**
### Overview

Test this on your local as a first step and if we are happy lets try the ci

### Designs

_Link to the related design prototypes (if applicable)_

### Testing instructions

_Please explain how to test the PR: ID of a dataset, steps to reach the feature, etc._

### Feature relevant tickets

_Link to the related task manager tickets_

---

## Checklist before submitting

- [ ] Meaningful commits and code rebased on `develop`.
- [ ] If this PR adds feature that should be tested for regressions when
      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)
- [ ] Update CHANGELOG file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -93,37 +112,28 @@ generate-geo-test-data: extract-geo-test-data
...
-		done;
-	SCENARIOID=$(shell docker-compose -f docker-compose-test-e2e.yml -f docker-compose-test-e2e.local.yml --env-file .env-test-e2e exec -T test-e2e-postgresql-api psql -X -A -t -U "${API_POSTGRES_USER}" -c "select id from scenarios where name = 'Example scenario 1 Project 1 Org 1'"); \
-	echo "appending data for scenario with id $${SCENARIOID}"; \
-	sed -e "s/\$$user/00000000-0000-0000-0000-000000000000/g" -e "s/\$$scenario/$$SCENARIOID/g" api/test/fixtures/test-geodata.sql | docker-compose -f docker-compose-test-e2e.yml -f docker-compose-test-e2e.local.yml --env-file .env-test-e2e exec -T test-e2e-postgresql-geo-api psql -U "${GEO_POSTGRES_USER}";
-	# run tests
-	docker-compose -f docker-compose-test-e2e.yml -f docker-compose-test-e2e.local.yml --env-file .env-test-e2e up --abort-on-container-exit --exit-code-from api api
-	# teardown
-	docker-compose -f docker-compose-test-e2e.yml -f docker-compose-test-e2e.local.yml --env-file .env-test-e2e rm --stop --force
-
+seed-test-dbs-instances:
```
Comment (aagm) [inline Makefile:115]: i think i can even delete this as the environment is already set and a concatenation of actions should be more than enough

File: Makefile [hunk 2]
```diff
@@ -93,37 +112,28 @@ generate-geo-test-data: extract-geo-test-data
...
+test-start-services: test-clean-slate
+	@echo "$(RED)Mounting docker file:$(NC) docker-compose-test-e2e.yml / docker-compose-test-e2e.local.yml"
+	# start from clean slate, in case anything was left around from previous runs (mostly relevant locally, not in CI) and spin the instances (geoprocessing, api and the DBs)
+	docker-compose $(DOCKER_COMPOSE_FILE) --project-name ${COMPOSE_PROJECT_NAME} up -d --build api geoprocessing && \
+	docker-compose $(DOCKER_COMPOSE_FILE) exec -T geoprocessing ./entrypoint.sh run-migrations-for-e2e-tests && \
+	docker-compose $(DOCKER_COMPOSE_FILE) exec -T api ./entrypoint.sh run-migrations-for-e2e-tests
+
+test-e2e-api: test-start-services | seed-test-dbs-instances
+	# run tests and remove containers
+	-docker-compose $(DOCKER_COMPOSE_FILE) exec -T api ./entrypoint.sh test-e2e
```
Comment (hotzevzl) [inline Makefile:128]: n00b q - why ignore non-zero exit codes?

Comment (hotzevzl) [reply Makefile:128]: ok, I get it :)

File: Makefile (hunk 1 above)
Comment (hotzevzl) [reply Makefile:115]: agree. keep it in mind for a cleanup PR 😛 I'm going to merge this in a moment 🚀

**Review by hotzevzl — APPROVED** [review]

**Pull request merged**